File: program/program/check-pangram-string/check_pangram_string.dart
bool isPangram(String text){
  Set<String> letters = "abcdefghijklmnopqrstuvwxyz".split("").toSet();
  text.toLowerCase().split("").forEach((l) => letters.remove(l));
  return letters.length == 0;
}

void main() {
  print(isPangram("The quick brown fox jumps over the lazy dog"));
}

File: program/program/check-pangram-string/CheckPangramString.swift
import Swift
import Foundation

func CheckPangram(str: String) -> Bool {
   let lowercaseString = str.lowercased()
   let alphabet = "abcdefghijklmnopqrstuvwxyz" 
   for l in alphabet {
      if !lowercaseString.contains(l) {
         return false
      }
   }
   return true
}

let str1 = "The quick brown fox jumps over the lazy dog"
print("'\(str1)' is pangram?:", CheckPangram(str: str1))
